src/main/java/edu/unc/lib/boxc/migration/cdm/services/ListProjectsService.java

```diff
@@ -80,13 +82,15 @@ private void listProjectsInDirectory(Path directory, ObjectMapper mapper, ArrayN
                         allowedActions = mapper.valueToTree(allowedActions(project));
                     }
                     JsonNode projectProperties = mapper.readTree(project.getProjectPropertiesPath().toFile());
+                    ObjectNode processingJobs = processingJobs(project, mapper);
 
                     // add project info to JSON
                     ObjectNode objectNode = mapper.createObjectNode();
                     objectNode.put(PROJECT_PATH, projectPath.toString());
                     objectNode.put(STATUS, projectStatus);
                     objectNode.putArray(ALLOWED_ACTIONS).addAll(allowedActions);
                     objectNode.set("projectProperties", projectProperties);
+                    objectNode.set("processingJobs", processingJobs);
```

Comment:
might as well make constants for both of these, since all the other top level keys have them